sdk/src/main/java/com/mapbox/maps/MapboxMap.kt

```diff
@@ -460,7 +462,7 @@ class MapboxMap internal constructor(
    * @return Returns the converted camera options
    */
   override fun cameraForCoordinates(
-    coordinates: List<Point>,
+    coordinates: List<Point?>,
```

Comment:
Why this has become nullable?

Reply:
When I copied the content form MapCameraManagerInterface into MapCameraManagerDelegate, it changes to nullable automatically.

Reply:
From the definition in MapCameraManagerInterface  `CameraOptions cameraForCoordinates(@NonNull List<Point> coordinates, @NonNull CameraOptions camera, @NonNull ScreenBox box);` it could accept a list with nullable content, but it seems make no sense to nullable. Will rollback this part.